e2e-test/trace/src/integrationTest/java/org/opensearch/dataprepper/integration/trace/EndToEndRawSpanTest.java

```diff
@@ -263,7 +263,7 @@ private List<Map<String, Object>> getExpectedDocuments(ExportTraceServiceRequest
         for(int i=0; i<exportTraceServiceRequests.length; i++) {
             exportTraceServiceRequests[i].getResourceSpansList().forEach( resourceSpans -> {
                 final String resourceName = getServiceName(resourceSpans);
-                resourceSpans.getInstrumentationLibrarySpansList().forEach( instrumentationLibrarySpans -> {
+                resourceSpans.getScopeSpansList().forEach( instrumentationLibrarySpans -> {
                     instrumentationLibrarySpans.getSpansList().forEach(span -> {
                         expectedDocuments.add(getExpectedEsDocumentSource(span, resourceName));
                     });
```

Comment:
Can you rename the local variable?
```suggestion
                resourceSpans.getScopeSpansList().forEach( scopeSpans -> {
                    scopeSpans.getSpansList().forEach(span -> {
                        expectedDocuments.add(getExpectedEsDocumentSource(span, resourceName));
                    });
```